BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/TelemetryConfigurationSetExporterOptionsTests.cs

```diff
@@ -0,0 +1,350 @@
+namespace Microsoft.ApplicationInsights.Tests
+{
+    using System;
+    using System.Reflection;
+    using Azure.Core;
+    using Azure.Monitor.OpenTelemetry.Exporter;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Options;
+    using OpenTelemetry;
+    using Xunit;
+
+    /// <summary>
+    /// Tests for Azure Monitor Exporter configuration methods in TelemetryConfiguration.
+    /// </summary>
+    public class TelemetryConfigurationSetExporterOptionsTests : IDisposable
+    {
+        private TelemetryConfiguration telemetryConfiguration;
+
+        public TelemetryConfigurationSetExporterOptionsTests()
+        {
+            this.telemetryConfiguration = new TelemetryConfiguration();
+        }
+
+        public void Dispose()
+        {
+            this.telemetryConfiguration?.Dispose();
+        }
+
+        /// <summary>
+        /// Helper method to get AzureMonitorExporterOptions from the built OpenTelemetrySdk.
+        /// </summary>
+        private AzureMonitorExporterOptions GetExporterOptions(OpenTelemetrySdk sdk)
+        {
+            // Use reflection to access the internal Services property
+            var servicesProperty = typeof(OpenTelemetrySdk).GetProperty(
+                "Services",
+                BindingFlags.NonPublic | BindingFlags.Instance);
+
+            var serviceProvider = servicesProperty?.GetValue(sdk) as IServiceProvider;
+            Assert.NotNull(serviceProvider);
+
+            var options = serviceProvider.GetService<IOptions<AzureMonitorExporterOptions>>();
+            Assert.NotNull(options);
+
+            return options.Value;
+        }
+
+        /// <summary>
+        /// Helper method to build the configuration by creating a TelemetryClient.
+        /// Creating a TelemetryClient triggers the Build() method internally.
+        /// </summary>
+        private OpenTelemetrySdk BuildConfiguration()
+        {
+            // Creating a TelemetryClient triggers configuration.Build() internally
+            var client = new TelemetryClient(this.telemetryConfiguration);
+
+            // Access the private 'sdk' field from TelemetryClient via reflection
+            var sdkField = typeof(TelemetryClient).GetField(
+                "sdk",
+                BindingFlags.NonPublic | BindingFlags.Instance);
+
+            return sdkField?.GetValue(client) as OpenTelemetrySdk;
+        }
+
+        #region SetSamplingRatio Tests
+
+        [Fact]
+        public void SetSamplingRatio_WithValidRatio_SetsSamplingRatioInExporterOptions()
+        {
+            // Arrange
+            this.telemetryConfiguration.ConnectionString = "InstrumentationKey=test-ikey";
+
+            // Act
+            this.telemetryConfiguration.SetSamplingRatio(0.5F);
```

Comment:
Could you set all six public APIs (introduced in this PR) in a single test and verify that their values are correctly applied? We need to make sure that the builder.Services.Configure<AzureMonitorExporterOptions> call does not overwrite them with default values.